### PR TITLE
Added many CF Constants (including Simple Geometry Ones)

### PR DESCRIPTION
**Pull request by wchen329**
Many new CF constants were added. Also added very small entries for Line and Polygon FTs (but not yet functional). Builds successfully.

### CODE REVIEW

File: cdm/src/main/java/ucar/nc2/constants/CF.java [hunk 1]
```diff
@@ -87,7 +142,7 @@
   public static final String ocean_double_sigma_coordinate = "ocean_double_sigma_coordinate";
 
   public static final String formula_terms = "formula_terms";
-
+
```
Comment (skaymen) [inline cdm/src/main/java/ucar/nc2/constants/CF.java]: I think some extra spaces snuck in here.

File: cdm/src/main/java/ucar/nc2/constants/CF.java [hunk 2]
```diff
@@ -114,7 +169,7 @@
 
   // not official - used in CFwriter for attribute name
   public static final String DSG_REPRESENTATION = "DSG_representation";
-
+
```
Comment (skaymen) [inline cdm/src/main/java/ucar/nc2/constants/CF.java]: Same thing may have happened here.

File: cdm/src/main/java/ucar/nc2/constants/FeatureType.java [hunk 3]
```diff
@@ -82,14 +85,15 @@ public static FeatureType getType(String name) {
...
+            (this == FeatureType.PROFILE) || (this == FeatureType.STATION_PROFILE) || (this == FeatureType.TRAJECTORY_PROFILE) ||
+            (this == FeatureType.LINE) || (this == FeatureType.POLYGON);
   }
 
   public boolean isCoverageFeatureType() {
     return (this == FeatureType.COVERAGE) ||
             (this == FeatureType.GRID) || (this == FeatureType.FMRC) || (this == FeatureType.SWATH)|| (this == FeatureType.CURVILINEAR);
   }
-
+
```
Comment (skaymen) [inline cdm/src/main/java/ucar/nc2/constants/FeatureType.java]: Also here.

**Review by skaymen — CHANGES_REQUESTED** [review]
Besides some extra spaces that sneaked in, everything looks fine to me.

**Comment by wchen329** [comment]
Cleaned up some unnecessary stuff based on good feedback

File: cdm/src/main/java/ucar/nc2/constants/FeatureType.java [hunk 4]
```diff
@@ -52,6 +53,8 @@
 
   ANY_POINT,  // Any of the point types
   POINT,      // unconnected points
+  LINE,		  // two or more connected points
+  POLYGON,	  // three or more connected lines which enclose an area
```
Comment (cschroedl-gov) [inline cdm/src/main/java/ucar/nc2/constants/FeatureType.java]: It's possible we will need to expand this to include Multiline and Multipolygon in the future. I think this is a good for now though.

**Review by cschroedl-gov — APPROVED** [review]
Looks good to me!

**Review by skaymen — APPROVED** [review]

File: cdm/src/main/java/ucar/nc2/constants/FeatureType.java [hunk 5]
```diff
@@ -49,6 +50,7 @@
   FMRC,       // two time dimensions, runtime and forecast time
   SWATH,      // 2D latlon, dependent time, polar orbiting satellites
   CURVILINEAR,// 2D latlon, independent time
+  SIMPGEO,	  // new simple geometry coverage type
```
Comment (cschroedl-gov) [inline cdm/src/main/java/ucar/nc2/constants/FeatureType.java]: With the exception of FMRC and UGRID, most constants in this file spell out words in their entirety. Should this enum entry be exceptional as well? If so, why?

Comment (wchen329) [reply cdm/src/main/java/ucar/nc2/constants/FeatureType.java]: Perhaps SIMPLEGEOMETRY would be monstrously long? Though if that is not the case it wouldn't be too difficult to change it to SIMPLEGEOMETRY.

Comment (wchen329) [reply cdm/src/main/java/ucar/nc2/constants/FeatureType.java]: Alright. Changing it to SIMPLE_GEOMETRY.

**Comment by wchen329** [comment]
Alright, changes made per good feedback.

**Review by cschroedl-gov — APPROVED** [review]

**Comment by wchen329** [comment]
This should be the final version

**Pull request merged**